qiskit/tools/monitor/job_monitor.py

```diff
@@ -79,6 +79,22 @@ def job_monitor(job, interval=None, quiet=False, output=sys.stdout, line_discipl
         By default this is sys.stdout.
         line_discipline (string): character emitted at start of a line of job monitor output,
         This defaults to \\r.
+
+    .. code-block:: python
+
+       from qiskit import BasicAer, transpile
+       from qiskit.circuit import QuantumRegister, ClassicalRegister, QuantumCircuit
+       from qiskit.tools.monitor import job_monitor
+       sim_backend = BasicAer.get_backend("qasm_simulator")
+       q = QuantumRegister(2)
+       c = ClassicalRegister(2)
+       qc = QuantumCircuit(q, c)
+       qc.h(q[0])
+       qc.cx(q[0], q[1])
+       qc.measure(q, c)
```

Comment:
Maybe this would be more up to date:
```suggestion
       from qiskit.circuit import QuantumCircuit
       from qiskit.tools.monitor import job_monitor
       sim_backend = BasicAer.get_backend("qasm_simulator")
       qc = QuantumCircuit(2, 2)
       qc.h(0)
       qc.cx(0, 1)
       qc.measure_all()
```